.zuul.d/jobs.yaml

```diff
@@ -46,3 +46,16 @@
     files:
       - github/projects.yaml
       - tools/manage-projects.py
+
+- job:
+    name: windmill-config-tox-tools
+    parent: tox
+    description: |
+      Runs the scripts from the tools directory
+
+      Uses tox with the ``tools`` environment.
+    vars:
+      tox_envlist: tools
+      test_setup_skip: true
+    files:
+      - nodepool/*.zuul.ansible.com.yaml
```

Comment:
should be regex, so

^nodepool

should be enough

Reply:
we also want to add tox.ini, as that is entry point and your tools script here.  So we test when both are also changed